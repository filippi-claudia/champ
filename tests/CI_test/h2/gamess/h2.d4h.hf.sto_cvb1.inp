 $SYSTEM MWORDS=90 $END
 $CONTRL RUNTYP=ENERGY SCFTYP=RHF DFTTYP=NONE MULT=1 
         MAXIT=200 COORD=UNIQUE UNITS=BOHR 
         ISPHER=1
 $END
 $DATA
H2 
DNH 4

H 1.0 0.00000000 0.00000000  0.70000000
1S 14 2.473106 
1S 14 1.165255 
1S 14 1.054968 
2P 14 1.481937

 $END 
