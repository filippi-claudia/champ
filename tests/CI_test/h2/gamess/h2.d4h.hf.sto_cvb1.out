----- GAMESS execution script 'rungms' -----
This job is running on host ccpgate
under operating system Linux at Wed Mar 11 11:12:23 CET 2020
Available scratch disk space (Kbyte units) at beginning of the job is
Filesystem       1K-blocks       Used   Available Use% Mounted on
/dev/sda2      17395693524 2406352288 14116012648  15% /home/filippi
GAMESS temporary binary files will be written to .
GAMESS supplementary output files will be written to .
Copying input file h2.d4h.hf.sto_cvb1.inp to your run's scratch directory...
cp -ip h2.d4h.hf.sto_cvb1.inp ./h2.d4h.hf.sto_cvb1.F05
unset echo
/home/filippi/AE_cusps/gamess14r1_modified/ddikick.x /home/filippi/AE_cusps/gamess14r1_modified/gamess_modified.00.x h2.d4h.hf.sto_cvb1 -ddi 1 1 ccpgate -scr .

 Distributed Data Interface kickoff program.
 Initiating 1 compute processes on 1 nodes to run the following command:
 /home/filippi/AE_cusps/gamess14r1_modified/gamess_modified.00.x h2.d4h.hf.sto_cvb1 

          ******************************************************
          *         GAMESS VERSION =  5 DEC 2014 (R1)          *
          *             FROM IOWA STATE UNIVERSITY             *
          * M.W.SCHMIDT, K.K.BALDRIDGE, J.A.BOATZ, S.T.ELBERT, *
          *   M.S.GORDON, J.H.JENSEN, S.KOSEKI, N.MATSUNAGA,   *
          *          K.A.NGUYEN, S.J.SU, T.L.WINDUS,           *
          *       TOGETHER WITH M.DUPUIS, J.A.MONTGOMERY       *
          *         J.COMPUT.CHEM.  14, 1347-1363(1993)        *
          **************** 64 BIT LINUX VERSION ****************

  SINCE 1993, STUDENTS AND POSTDOCS WORKING AT IOWA STATE UNIVERSITY
  AND ALSO IN THEIR VARIOUS JOBS AFTER LEAVING ISU HAVE MADE IMPORTANT
  CONTRIBUTIONS TO THE CODE:
     IVANA ADAMOVIC, CHRISTINE AIKENS, YURI ALEXEEV, POOJA ARORA,
     ANDREY ASADCHEV, ROB BELL, PRADIPTA BANDYOPADHYAY, JONATHAN BENTZ,
     BRETT BODE, KURT BRORSEN, CALEB CARLIN, GALINA CHABAN, WEI CHEN,
     CHEOL HO CHOI, PAUL DAY, ALBERT DEFUSCO, NUWAN DESILVA, TIM DUDLEY,
     DMITRI FEDOROV, GRAHAM FLETCHER, MARK FREITAG, KURT GLAESEMANN,
     DAN KEMP, GRANT MERRILL, NORIYUKI MINEZAWA, JONATHAN MULLIN,
     TAKESHI NAGATA, SEAN NEDD, HEATHER NETZLOFF, BOSILJKA NJEGIC, RYAN OLSON,
     MIKE PAK, SPENCER PRUITT, LUKE ROSKOP, JIM SHOEMAKER, LYUDMILA SLIPCHENKO,
     TONY SMITH, SAROM SOK, JIE SONG, TETSUYA TAKETSUGU, SIMON WEBB,
     PENG XU, SOOHAENG YOO, FEDERICO ZAHARIEV

  ADDITIONAL CODE HAS BEEN PROVIDED BY COLLABORATORS IN OTHER GROUPS:
     IOWA STATE UNIVERSITY:
          JOE IVANIC, AARON WEST, LAIMUTIS BYTAUTAS, KLAUS RUEDENBERG
     UNIVERSITY OF TOKYO: KIMIHIKO HIRAO, TAKAHITO NAKAJIMA,
          TAKAO TSUNEDA, MUNEAKI KAMIYA, SUSUMU YANAGISAWA,
          KIYOSHI YAGI, MAHITO CHIBA, SEIKEN TOKURA, NAOAKI KAWAKAMI
     UNIVERSITY OF AARHUS: FRANK JENSEN
     UNIVERSITY OF IOWA: VISVALDAS KAIRYS, HUI LI
     NATIONAL INST. OF STANDARDS AND TECHNOLOGY: WALT STEVENS, DAVID GARMER
     UNIVERSITY OF PISA: BENEDETTA MENNUCCI, JACOPO TOMASI
     UNIVERSITY OF MEMPHIS: HENRY KURTZ, PRAKASHAN KORAMBATH
     UNIVERSITY OF ALBERTA: TOBY ZENG, MARIUSZ KLOBUKOWSKI
     UNIVERSITY OF NEW ENGLAND: MARK SPACKMAN
     MIE UNIVERSITY: HIROAKI UMEDA
     NAT. INST. OF ADVANCED INDUSTRIAL SCIENCE AND TECHNOLOGY: KAZUO KITAURA
     MICHIGAN STATE UNIVERSITY:
          KAROL KOWALSKI, MARTA WLOCH, JEFFREY GOUR, JESSE LUTZ,
          WEI LI, PIOTR PIECUCH
     UNIVERSITY OF SILESIA: MONIKA MUSIAL, STANISLAW KUCHARSKI
     FACULTES UNIVERSITAIRES NOTRE-DAME DE LA PAIX:
          OLIVIER QUINET, BENOIT CHAMPAGNE
     UNIVERSITY OF CALIFORNIA - SANTA BARBARA: BERNARD KIRTMAN
     INSTITUTE FOR MOLECULAR SCIENCE:
          KAZUYA ISHIMURA, MICHIO KATOUDA, AND SHIGERU NAGASE
     UNIVERSITY OF NOTRE DAME: ANNA POMOGAEVA, DAN CHIPMAN
     KYUSHU UNIVERSITY:
          HARUYUKI NAKANO,
          FENG LONG GU, JACEK KORCHOWIEC, MARCIN MAKOWSKI, AND YURIKO AOKI,
          HIROTOSHI MORI AND EISAKU MIYOSHI
     PENNSYLVANIA STATE UNIVERSITY:
          TZVETELIN IORDANOV, CHET SWALINA, JONATHAN SKONE,
          SHARON HAMMES-SCHIFFER
     WASEDA UNIVERSITY:
          MASATO KOBAYASHI, TOMOKO AKAMA, TSUGUKI TOUMA,
          TAKESHI YOSHIKAWA, YASUHIRO IKABATA, HIROMI NAKAI
     NANJING UNIVERSITY: SHUHUA LI
     UNIVERSITY OF NEBRASKA:
          PEIFENG SU, DEJUN SI, NANDUN THELLAMUREGE, YALI WANG, HUI LI
     UNIVERSITY OF ZURICH: ROBERTO PEVERATI, KIM BALDRIDGE
     N. COPERNICUS UNIVERSITY AND JACKSON STATE UNIVERSITY:
          MARIA BARYSZ
     UNIVERSITY OF COPENHAGEN: CASPER STEINMANN
     TOKYO INSTITUTE OF TECHNOLOGY: HIROYA NAKATA
     NAGOYA UNIVERSITY: YOSHIO NISHIMOTO, STEPHAN IRLE

 EXECUTION OF GAMESS BEGUN Wed Mar 11 11:12:23 2020

            ECHO OF THE FIRST FEW INPUT CARDS -
 INPUT CARD> $SYSTEM MWORDS=90 $END                                                         
 INPUT CARD> $CONTRL RUNTYP=ENERGY SCFTYP=RHF DFTTYP=NONE MULT=1                            
 INPUT CARD>         MAXIT=200 COORD=UNIQUE UNITS=BOHR                                      
 INPUT CARD>         ISPHER=1                                                               
 INPUT CARD> $END                                                                           
 INPUT CARD> $DATA                                                                          
 INPUT CARD>H2                                                                              
 INPUT CARD>DNH 4                                                                           
 INPUT CARD>                                                                                
 INPUT CARD>H 1.0 0.00000000 0.00000000  0.70000000                                         
 INPUT CARD>1S 14 2.473106                                                                  
 INPUT CARD>1S 14 1.165255                                                                  
 INPUT CARD>1S 14 1.054968                                                                  
 INPUT CARD>2P 14 1.481937                                                                  
 INPUT CARD>                                                                                
 INPUT CARD> $END                                                                           
   90000000 WORDS OF MEMORY AVAILABLE


     RUN TITLE
     ---------
 H2                                                                              

 THE POINT GROUP OF THE MOLECULE IS DNH     
 THE ORDER OF THE PRINCIPAL AXIS IS     4

 ATOM      ATOMIC                      COORDINATES (BOHR)
           CHARGE         X                   Y                   Z
 H           1.0     0.0000000000        0.0000000000       -0.7000000000
 H           1.0     0.0000000000        0.0000000000        0.7000000000

          INTERNUCLEAR DISTANCES (ANGS.)
          ------------------------------

                1 H          2 H     

   1 H       0.0000000    0.7408481 *
   2 H       0.7408481 *  0.0000000  

  * ... LESS THAN  3.000

 ATOMS: STO-NG TYPE BASIS SET

     ATOMIC BASIS SET
     ----------------
 THE CONTRACTED PRIMITIVE FUNCTIONS HAVE BEEN UNNORMALIZED
 THE CONTRACTED BASIS FUNCTIONS ARE NOW NORMALIZED TO UNITY

 SHELL TYPE PRIMITIVE    SLATER EXPONENT     EXPONENT          CONTRACTION COEFFICIENTS

 H         

   5   1S      1             2.4731060         13126.6412842      0.000032071854
   5   1S      2                                2411.0392530      0.000180868724
   5   1S      3                                 675.5834605      0.000715732871
   5   1S      4                                 232.8436376      0.002327137410
   5   1S      5                                  91.3577887      0.006621033466
   5   1S      6                                  39.2633689      0.016968581946
   5   1S      7                                  18.0718102      0.039565969540
   5   1S      8                                   8.7792496      0.083424650726
   5   1S      9                                   4.4560303      0.155102805406
   5   1S     10                                   2.3452086      0.241346852708
   5   1S     11                                   1.2717867      0.286294283519
   5   1S     12                                   0.7059528      0.220624522944
   5   1S     13                                   0.3969666      0.083307088014
   5   1S     14                                   0.2199573      0.008488240712

   6   1S     15             1.1652550          2914.1379415      0.000032071854
   6   1S     16                                 535.2550446      0.000180868724
   6   1S     17                                 149.9807416      0.000715732871
   6   1S     18                                  51.6917058      0.002327137410
   6   1S     19                                  20.2815932      0.006621033466
   6   1S     20                                   8.7165384      0.016968581946
   6   1S     21                                   4.0119743      0.039565969540
   6   1S     22                                   1.9490092      0.083424650726
   6   1S     23                                   0.9892467      0.155102805406
   6   1S     24                                   0.5206405      0.241346852708
   6   1S     25                                   0.2823389      0.286294283519
   6   1S     26                                   0.1567228      0.220624522944
   6   1S     27                                   0.0881273      0.083307088014
   6   1S     28                                   0.0488309      0.008488240712

   7   1S     29             1.0549680          2388.6181510      0.000032071854
   7   1S     30                                 438.7300603      0.000180868724
   7   1S     31                                 122.9340302      0.000715732871
   7   1S     32                                  42.3699046      0.002327137410
   7   1S     33                                  16.6241210      0.006621033466
   7   1S     34                                   7.1446453      0.016968581946
   7   1S     35                                   3.2884767      0.039565969540
   7   1S     36                                   1.5975355      0.083424650726
   7   1S     37                                   0.8108514      0.155102805406
   7   1S     38                                   0.4267510      0.241346852708
   7   1S     39                                   0.2314235      0.286294283519
   7   1S     40                                   0.1284603      0.220624522944
   7   1S     41                                   0.0722349      0.083307088014
   7   1S     42                                   0.0400250      0.008488240712

   8   2P     43             1.4819370           502.0172226      0.000013589949
   8   2P     44                                 131.1667163      0.000098737858
   8   2P     45                                  47.1203199      0.000479801845
   8   2P     46                                  19.8636220      0.001857311798
   8   2P     47                                   9.2664024      0.006124920625
   8   2P     48                                   4.6441308      0.017694065197
   8   2P     49                                   2.4571228      0.045024238144
   8   2P     50                                   1.3567189      0.099589645015
   8   2P     51                                   0.7754408      0.185117675279
   8   2P     52                                   0.4558807      0.272215006419
   8   2P     53                                   0.2741232      0.287038979472
   8   2P     54                                   0.1675169      0.185359431716
   8   2P     55                                   0.1029730      0.055951602281
   8   2P     56                                   0.0620189      0.004428704232

 TOTAL NUMBER OF BASIS SET SHELLS             =    8
 NUMBER OF CARTESIAN GAUSSIAN BASIS FUNCTIONS =   12
 NOTE: THIS RUN WILL RESTRICT THE MO VARIATION SPACE TO SPHERICAL HARMONICS.
 THE NUMBER OF ORBITALS KEPT IN THE VARIATIONAL SPACE WILL BE PRINTED LATER.
 NUMBER OF ELECTRONS                          =    2
 CHARGE OF MOLECULE                           =    0
 SPIN MULTIPLICITY                            =    1
 NUMBER OF OCCUPIED ORBITALS (ALPHA)          =    1
 NUMBER OF OCCUPIED ORBITALS (BETA )          =    1
 TOTAL NUMBER OF ATOMS                        =    2
 THE NUCLEAR REPULSION ENERGY IS        0.7142857143

 THIS MOLECULE IS RECOGNIZED AS BEING LINEAR,
 ORBITAL LZ DEGENERACY TOLERANCE ETOLLZ= 1.00E-06

     $CONTRL OPTIONS
     ---------------
 SCFTYP=RHF          RUNTYP=ENERGY       EXETYP=RUN     
 MPLEVL=       0     CITYP =NONE         CCTYP =NONE         VBTYP =NONE    
 DFTTYP=NONE         TDDFT =NONE    
 MULT  =       1     ICHARG=       0     NZVAR =       0     COORD =UNIQUE  
 PP    =NONE         RELWFN=NONE         LOCAL =NONE         NUMGRD=       F
 ISPHER=       1     NOSYM =       0     MAXIT =     200     UNITS =BOHR    
 PLTORB=       F     MOLPLT=       F     AIMPAC=       F     FRIEND=        
 NPRINT=       7     IREST =       0     GEOM  =INPUT   
 NORMF =       0     NORMP =       0     ITOL  =      20     ICUT  =       9
 INTTYP=BEST         GRDTYP=BEST         QMTTOL= 1.0E-06

     $SYSTEM OPTIONS
     ---------------
  REPLICATED MEMORY=    90000000 WORDS (ON EVERY NODE).
 DISTRIBUTED MEMDDI=           0 MILLION WORDS IN AGGREGATE,
 MEMDDI DISTRIBUTED OVER   1 PROCESSORS IS           0 WORDS/PROCESSOR.
 TOTAL MEMORY REQUESTED ON EACH PROCESSOR=    90000000 WORDS.
 TIMLIM=      525600.00 MINUTES, OR     365.0 DAYS.
 PARALL= F  BALTYP=  DLB     KDIAG=    0  COREFL= F
 MXSEQ2=     300 MXSEQ3=     150  mem10=         0

          ----------------
          PROPERTIES INPUT
          ----------------

     MOMENTS            FIELD           POTENTIAL          DENSITY
 IEMOM =       1   IEFLD =       0   IEPOT =       0   IEDEN =       0
 WHERE =COMASS     WHERE =NUCLEI     WHERE =NUCLEI     WHERE =NUCLEI  
 OUTPUT=BOTH       OUTPUT=BOTH       OUTPUT=BOTH       OUTPUT=BOTH    
 IEMINT=       0   IEFINT=       0                     IEDINT=       0
                                                       MORB  =       0
          EXTRAPOLATION IN EFFECT
          DIIS IN EFFECT
 ORBITAL PRINTING OPTION: NPREO=     1    12     2     1

     -------------------------------
     INTEGRAL TRANSFORMATION OPTIONS
     -------------------------------
     NWORD  =            0
     CUTOFF = 1.0E-09     MPTRAN =       0
     DIRTRF =       F     AOINTS =DUP     

          ----------------------
          INTEGRAL INPUT OPTIONS
          ----------------------
 NOPK  =       1 NORDER=       0 SCHWRZ=       F

     ------------------------------------------
     THE POINT GROUP IS DNH, NAXIS= 4, ORDER=16
     ------------------------------------------

 -- VARIATIONAL SPACE WILL BE RESTRICTED TO PURE SPHERICAL HARMONICS ONLY --
 AFTER EXCLUDING CONTAMINANT COMBINATIONS FROM THE CARTESIAN GAUSSIAN BASIS
 SET, THE NUMBER OF SPHERICAL HARMONICS KEPT IN THE VARIATION SPACE IS   12

     DIMENSIONS OF THE SYMMETRY SUBSPACES ARE
 A1G =    4     A1U =    0     B1G =    0     B1U =    0     A2G =    0
 A2U =    4     B2G =    0     B2U =    0     EG  =    1     EU  =    1

 ..... DONE SETTING UP THE RUN .....
 STEP CPU TIME =     0.02 TOTAL CPU TIME =        0.0 (    0.0 MIN)
 TOTAL WALL CLOCK TIME=        0.0 SECONDS, CPU UTILIZATION IS 200.00%

          ********************
          1 ELECTRON INTEGRALS
          ********************
 ...... END OF ONE-ELECTRON INTEGRALS ......
 STEP CPU TIME =     0.00 TOTAL CPU TIME =        0.0 (    0.0 MIN)
 TOTAL WALL CLOCK TIME=        0.0 SECONDS, CPU UTILIZATION IS 100.00%

          -------------
          GUESS OPTIONS
          -------------
          GUESS =HUCKEL            NORB  =       0          NORDER=       0
          MIX   =       F          PRTMO =       F          PUNMO =       F
          TOLZ  = 1.0E-08          TOLE  = 1.0E-05
          SYMDEN=       F          PURIFY=       F

 INITIAL GUESS ORBITALS GENERATED BY HUCKEL   ROUTINE.
 HUCKEL GUESS REQUIRES      2468 WORDS.

 SYMMETRIES FOR INITIAL GUESS ORBITALS FOLLOW.   BOTH SET(S).
     1 ORBITALS ARE OCCUPIED (    0 CORE ORBITALS).
     1=A1G      2=A2U      3=A1G      4=A1G      5=A1G      6=A2U      7=A2U 
     8=A2U      9=EG      10=EG      11=EU  
 ...... END OF INITIAL ORBITAL SELECTION ......
 STEP CPU TIME =     0.00 TOTAL CPU TIME =        0.0 (    0.0 MIN)
 TOTAL WALL CLOCK TIME=        0.0 SECONDS, CPU UTILIZATION IS 100.00%

                    ----------------------
                    AO INTEGRAL TECHNOLOGY
                    ----------------------
     S,P,L SHELL ROTATED AXIS INTEGRALS, REPROGRAMMED BY
        KAZUYA ISHIMURA (IMS) AND JOSE SIERRA (SYNSTAR).
     S,P,D,L SHELL ROTATED AXIS INTEGRALS PROGRAMMED BY
        KAZUYA ISHIMURA (INSTITUTE FOR MOLECULAR SCIENCE).
     S,P,D,F,G SHELL TO TOTAL QUARTET ANGULAR MOMENTUM SUM 5,
        ERIC PROGRAM BY GRAHAM FLETCHER (ELORET AND NASA ADVANCED
        SUPERCOMPUTING DIVISION, AMES RESEARCH CENTER).
     S,P,D,F,G,L SHELL GENERAL RYS QUADRATURE PROGRAMMED BY
        MICHEL DUPUIS (PACIFIC NORTHWEST NATIONAL LABORATORY).

          --------------------
          2 ELECTRON INTEGRALS
          --------------------

 THE -PK- OPTION IS OFF, THE INTEGRALS ARE NOT IN SUPERMATRIX FORM.
 STORING   15000 INTEGRALS/RECORD ON DISK, USING 12 BYTES/INTEGRAL.
 TWO ELECTRON INTEGRAL EVALUATION REQUIRES   89392 WORDS OF MEMORY.
 II,JST,KST,LST =  1  1  1  1 NREC =         1 INTLOC =    1
 II,JST,KST,LST =  2  1  1  1 NREC =         1 INTLOC =    1
 II,JST,KST,LST =  3  1  1  1 NREC =         1 INTLOC =    1
 II,JST,KST,LST =  4  1  1  1 NREC =         1 INTLOC =    1
 II,JST,KST,LST =  5  1  1  1 NREC =         1 INTLOC =    1
 II,JST,KST,LST =  6  1  1  1 NREC =         1 INTLOC =    5
 II,JST,KST,LST =  7  1  1  1 NREC =         1 INTLOC =   32
 II,JST,KST,LST =  8  1  1  1 NREC =         1 INTLOC =  124
 TOTAL NUMBER OF NONZERO TWO-ELECTRON INTEGRALS =                 594
          1 INTEGRAL RECORDS WERE STORED ON DISK FILE  8.
  ...... END OF TWO-ELECTRON INTEGRALS .....
 STEP CPU TIME =     0.26 TOTAL CPU TIME =        0.3 (    0.0 MIN)
 TOTAL WALL CLOCK TIME=        0.3 SECONDS, CPU UTILIZATION IS 100.00%

          --------------------------
                 RHF SCF CALCULATION
          --------------------------

     NUCLEAR ENERGY =         0.7142857143
     MAXIT =  200     NPUNCH=    2
     EXTRAP=T  DAMP=F  SHIFT=F  RSTRCT=F  DIIS=T  DEM=F  SOSCF=F
     DENSITY MATRIX CONV=  1.00E-05
     MEMORY REQUIRED FOR RHF ITERS=     51816 WORDS.

 ITER EX DEM     TOTAL ENERGY        E CHANGE  DENSITY CHANGE    DIIS ERROR
          * * *   INITIATING DIIS PROCEDURE   * * *
   1  0  0       -1.1222150794    -1.1222150794   2.085216371   0.103780404
   2  1  0       -1.1326655636    -0.0104504842   0.344577082   0.022630847
   3  2  0       -1.1331720321    -0.0005064685   0.004292987   0.000371721
   4  3  0       -1.1331721752    -0.0000001431   0.000353823   0.000026897
   5  4  0       -1.1331721760    -0.0000000008   0.000002244   0.000000221
   6  5  0       -1.1331721760    -0.0000000000   0.000000001   0.000000000

          -----------------
          DENSITY CONVERGED
          -----------------
     TIME TO FORM FOCK OPERATORS=       0.0 SECONDS (       0.0 SEC/ITER)
     TIME TO SOLVE SCF EQUATIONS=       0.0 SECONDS (       0.0 SEC/ITER)

 FINAL RHF ENERGY IS       -1.1331721760 AFTER   6 ITERATIONS

 LZ VALUE ANALYSIS FOR THE MOS
 ----------------------------------------
 MO     1 (    1) HAS LZ(WEIGHT)= 0.00(100.0%) 
 MO     2 (    2) HAS LZ(WEIGHT)= 0.00(100.0%) 
 MO     3 (    3) HAS LZ(WEIGHT)= 0.00(100.0%) 
 MO     4 (    4) HAS LZ(WEIGHT)=-1.00( 50.0%)  1.00( 50.0%) 
 MO     5 (    4) HAS LZ(WEIGHT)=-1.00( 50.0%)  1.00( 50.0%) 
 MO     6 (    5) HAS LZ(WEIGHT)= 0.00(100.0%) 
 MO     7 (    6) HAS LZ(WEIGHT)= 0.00(100.0%) 
 MO     8 (    7) HAS LZ(WEIGHT)=-1.00( 50.0%)  1.00( 50.0%) 
 MO     9 (    7) HAS LZ(WEIGHT)=-1.00( 50.0%)  1.00( 50.0%) 
 MO    10 (    8) HAS LZ(WEIGHT)= 0.00(100.0%) 
 MO    11 (    9) HAS LZ(WEIGHT)= 0.00(100.0%) 
 MO    12 (   10) HAS LZ(WEIGHT)= 0.00(100.0%) 

          ------------
          EIGENVECTORS
          ------------

                        1              2              3              4              5
                   -0.594322111    0.207308067    0.367783833    0.704458470    0.704458470
                       A1G            A2U            A1G            EU             EU  
    1  H  1  S      0.015709769    0.337495190   -0.454327934    0.000000000    0.000000000
    2  H  1  S      0.441002273  -12.974656251    9.105179335    0.000000000    0.000000000
    3  H  1  S      0.078689584   14.746577643   -8.707358186    0.000000000    0.000000000
    4  H  1  X     -0.000000000   -0.000000000    0.000000000    0.546003798    0.000000000
    5  H  1  Y     -0.000000000   -0.000000000    0.000000000    0.000000000    0.546003798
    6  H  1  Z      0.036505877    0.081330007    0.015706084    0.000000000    0.000000000
    7  H  2  S      0.015709769   -0.337495190   -0.454327934    0.000000000    0.000000000
    8  H  2  S      0.441002273   12.974656251    9.105179335    0.000000000    0.000000000
    9  H  2  S      0.078689584  -14.746577643   -8.707358186    0.000000000    0.000000000
   10  H  2  X     -0.000000000   -0.000000000    0.000000000    0.546003798    0.000000000
   11  H  2  Y     -0.000000000   -0.000000000    0.000000000    0.000000000    0.546003798
   12  H  2  Z     -0.036505877    0.081330007   -0.015706084    0.000000000    0.000000000

                        6              7              8              9             10
                    0.902135108    1.489862223    1.591064762    1.591064762    3.347424886
                       A2U            A1G            EG             EG             A2U 
    1  H  1  S      0.780572399   -0.397135945   -0.000000000    0.000000000   -0.352517075
    2  H  1  S    -22.487379050    5.171757929   -0.000000000    0.000000000  -10.222320893
    3  H  1  S     23.315619903   -4.509445086   -0.000000000    0.000000000   17.116138215
    4  H  1  X      0.000000000    0.000000000   -0.000000000    1.244520894    0.000000000
    5  H  1  Y      0.000000000    0.000000000    1.244520894    0.000000000    0.000000000
    6  H  1  Z      0.503468707   -0.953994772   -0.000000000    0.000000000    2.801633866
    7  H  2  S     -0.780572399   -0.397135945   -0.000000000    0.000000000    0.352517075
    8  H  2  S     22.487379050    5.171757929   -0.000000000    0.000000000   10.222320893
    9  H  2  S    -23.315619903   -4.509445086   -0.000000000    0.000000000  -17.116138215
   10  H  2  X      0.000000000    0.000000000   -0.000000000   -1.244520894    0.000000000
   11  H  2  Y      0.000000000    0.000000000   -1.244520894    0.000000000    0.000000000
   12  H  2  Z      0.503468707    0.953994772   -0.000000000    0.000000000    2.801633866

                       11             12
                    4.769798810    7.246219192
                       A1G            A2U 
    1  H  1  S     -2.005004499   -3.797669885
    2  H  1  S      9.570239936   18.947107604
    3  H  1  S     -8.075130130  -10.884573172
    4  H  1  X     -0.000000000   -0.000000000
    5  H  1  Y     -0.000000000   -0.000000000
    6  H  1  Z      0.099336348    2.282666130
    7  H  2  S     -2.005004499    3.797669885
    8  H  2  S      9.570239936  -18.947107604
    9  H  2  S     -8.075130130   10.884573172
   10  H  2  X     -0.000000000   -0.000000000
   11  H  2  Y     -0.000000000   -0.000000000
   12  H  2  Z     -0.099336348    2.282666130
 ...... END OF RHF CALCULATION ......
 STEP CPU TIME =     0.00 TOTAL CPU TIME =        0.3 (    0.0 MIN)
 TOTAL WALL CLOCK TIME=        0.3 SECONDS, CPU UTILIZATION IS 100.00%

     ----------------------------------------------------------------
     PROPERTY VALUES FOR THE RHF   SELF-CONSISTENT FIELD WAVEFUNCTION
     ----------------------------------------------------------------

          -----------------
          ENERGY COMPONENTS
          -----------------

         WAVEFUNCTION NORMALIZATION =       1.0000000000

                ONE ELECTRON ENERGY =      -2.5062715585
                TWO ELECTRON ENERGY =       0.6588136682
           NUCLEAR REPULSION ENERGY =       0.7142857143
                                      ------------------
                       TOTAL ENERGY =      -1.1331721760

 ELECTRON-ELECTRON POTENTIAL ENERGY =       0.6588136682
  NUCLEUS-ELECTRON POTENTIAL ENERGY =      -3.6309392637
   NUCLEUS-NUCLEUS POTENTIAL ENERGY =       0.7142857143
                                      ------------------
             TOTAL POTENTIAL ENERGY =      -2.2578398812
               TOTAL KINETIC ENERGY =       1.1246677053
                 VIRIAL RATIO (V/T) =       2.0075617631

  ...... PI ENERGY ANALYSIS ......

 ENERGY ANALYSIS:
            FOCK ENERGY=     -1.1886442221
          BARE H ENERGY=     -2.5062715585
     ELECTRONIC ENERGY =     -1.8474578903
         KINETIC ENERGY=      1.1246677053
          N-N REPULSION=      0.7142857143
           TOTAL ENERGY=     -1.1331721760
        SIGMA PART(1+2)=     -1.8474578903
               (K,V1,2)=      1.1246677053     -3.6309392637      0.6588136682
           PI PART(1+2)=      0.0000000000
               (K,V1,2)=      0.0000000000      0.0000000000      0.0000000000
  SIGMA SKELETON, ERROR=     -1.1331721760     -0.0000000000
             MIXED PART= 0.00000E+00 0.00000E+00 0.00000E+00 0.00000E+00
 ...... END OF PI ENERGY ANALYSIS ......

          ---------------------------------------
          MULLIKEN AND LOWDIN POPULATION ANALYSES
          ---------------------------------------

     ATOMIC MULLIKEN POPULATION IN EACH MOLECULAR ORBITAL

                      1

                  2.000000

    1             1.000000
    2             1.000000

               ----- POPULATIONS IN EACH AO -----
                             MULLIKEN      LOWDIN
              1  H  1  S      0.02105     0.13996
              2  H  1  S      0.80972     0.41442
              3  H  1  S      0.14567     0.41242
              4  H  1  X      0.00000     0.00000
              5  H  1  Y      0.00000     0.00000
              6  H  1  Z      0.02356     0.03320
              7  H  2  S      0.02105     0.13996
              8  H  2  S      0.80972     0.41442
              9  H  2  S      0.14567     0.41242
             10  H  2  X      0.00000     0.00000
             11  H  2  Y      0.00000     0.00000
             12  H  2  Z      0.02356     0.03320

          ----- MULLIKEN ATOMIC OVERLAP POPULATIONS -----
          (OFF-DIAGONAL ELEMENTS NEED TO BE MULTIPLIED BY 2)

             1           2

    1    0.5691194
    2    0.4308806   0.5691194

          TOTAL MULLIKEN AND LOWDIN ATOMIC POPULATIONS
       ATOM         MULL.POP.    CHARGE          LOW.POP.     CHARGE
    1 H             1.000000    0.000000         1.000000   -0.000000
    2 H             1.000000    0.000000         1.000000    0.000000

          MULLIKEN SPHERICAL HARMONIC POPULATIONS
       ATOM           S       P       D      F      G      H      I    TOTAL
    1 H             0.98    0.02    0.00   0.00   0.00   0.00   0.00    1.00
    2 H             0.98    0.02    0.00   0.00   0.00   0.00   0.00    1.00

          -------------------------------
          BOND ORDER AND VALENCE ANALYSIS     BOND ORDER THRESHOLD=0.050
          -------------------------------

                   BOND                       BOND                       BOND
  ATOM PAIR DIST  ORDER      ATOM PAIR DIST  ORDER      ATOM PAIR DIST  ORDER
    1   2  0.741  1.000

                       TOTAL       BONDED        FREE
      ATOM            VALENCE     VALENCE     VALENCE
    1 H                 1.000       1.000       0.000
    2 H                 1.000       1.000       0.000

          ---------------------
          ELECTROSTATIC MOMENTS
          ---------------------

 POINT   1           X           Y           Z (BOHR)    CHARGE
                 0.000000    0.000000    0.000000        0.00 (A.U.)
         DX          DY          DZ         /D/  (DEBYE)
     0.000000    0.000000    0.000000    0.000000
 ...... END OF PROPERTY EVALUATION ......
 STEP CPU TIME =     0.00 TOTAL CPU TIME =        0.3 (    0.0 MIN)
 TOTAL WALL CLOCK TIME=        0.3 SECONDS, CPU UTILIZATION IS  96.55%
               580000  WORDS OF DYNAMIC MEMORY USED
 EXECUTION OF GAMESS TERMINATED NORMALLY Wed Mar 11 11:12:24 2020
 DDI: 263640 bytes (0.3 MB / 0 MWords) used by master data server.

 ----------------------------------------
 CPU timing information for all processes
 ========================================
 0: 0.279 + 0.15 = 0.295
 ----------------------------------------
Note: The following floating-point exceptions are signalling: IEEE_UNDERFLOW_FLAG IEEE_DENORMAL
 ddikick.x: exited gracefully.
unset echo
----- accounting info -----
Files used on the master node ccpgate were:
-rw-rw-r-- 1 filippi filippi    3079 Mar 11 11:12 ./h2.d4h.hf.sto_cvb1.dat
-rw-rw-r-- 1 filippi filippi     275 Mar 11 11:10 ./h2.d4h.hf.sto_cvb1.F05
-rw-rw-r-- 1 filippi filippi  180016 Mar 11 11:12 ./h2.d4h.hf.sto_cvb1.F08
-rw-rw-r-- 1 filippi filippi 1439680 Mar 11 11:12 ./h2.d4h.hf.sto_cvb1.F10
-rw-rw-r-- 1 filippi filippi     275 Mar 11 11:10 ./h2.d4h.hf.sto_cvb1.inp
-rw-rw-r-- 1 filippi filippi   29153 Mar 11 11:12 ./h2.d4h.hf.sto_cvb1.out
ls: No match.
ls: No match.
ls: No match.
Wed Mar 11 11:12:27 CET 2020
0.237u 0.167s 0:03.73 10.4%	0+0k 0+16io 0pf+0w
